File: decompiled/3.16.1/be/ehealth/businessconnector/therlink/domain/responses/RevokeTherapeuticLinkResponse.java
package be.ehealth.businessconnector.therlink.domain.responses;

import be.ehealth.businessconnector.therlink.domain.Author;
import be.ehealth.businessconnector.therlink.domain.TherapeuticLinkRequestType;
import org.joda.time.DateTime;

public class RevokeTherapeuticLinkResponse extends GeneralResponse {
   public RevokeTherapeuticLinkResponse(DateTime dateTime, Author author, String id, TherapeuticLinkRequestType request, Acknowledge acknowledge) {
      this.dateTime = dateTime;
      this.externalAuthor = author;
      this.externalID = id;
      this.originalRequest = request;
      this.acknowledge = acknowledge;
   }
}

File: src/main/kotlin/org/taktik/connector/business/therlink/domain/responses/RevokeTherapeuticLinkResponse.kt
package org.taktik.connector.business.therlink.domain.responses

import org.taktik.connector.business.therlink.domain.Author
import org.taktik.connector.business.therlink.domain.TherapeuticLinkRequestType
import org.joda.time.DateTime

class RevokeTherapeuticLinkResponse(
    dateTime: DateTime,
    author: Author,
    id: String,
    request: TherapeuticLinkRequestType,
    acknowledge: Acknowledge
) : GeneralResponse(dateTime, author, id, request, acknowledge)
